src/OracleService/Protocols/OracleHttpsProtocol.cs

```diff
@@ -22,7 +22,10 @@ namespace Neo.Plugins
 {
     class OracleHttpsProtocol : IOracleProtocol
     {
-        private readonly HttpClient client = new(new HttpClientHandler() { AllowAutoRedirect = false });
+        private readonly HttpClient client = new(new HttpClientHandler() { AllowAutoRedirect = false })
+        {
+            MaxResponseContentBufferSize = OracleResponse.MaxResultSize
```

Comment:
We should probably adjust the exception handler of Oracle service wrt this change. Currently if the responce content exceeds `OracleResponse.MaxResultSize` length, the exception will be thrown by `message.Content.ReadAsStringAsync(...)` method:
https://github.com/neo-project/neo-modules/blob/4e310685d777c4d2b550d583a640849a9eb82403/src/OracleService/Protocols/OracleHttpsProtocol.cs#L85
It will be catched by the following `catch` block:
https://github.com/neo-project/neo-modules/blob/4e310685d777c4d2b550d583a640849a9eb82403/src/OracleService/OracleService.cs#L361-L368
As a result, we'll have `OracleResponseCode.Error` code instead of `OracleResponseCode.ResponseTooLarge` which is more appropriate in this case. In neo-go we return `OracleResponseCode.ResponseTooLarge` in this case.

Reply:
Which exception is throw?

Reply:
@superboyiii could you test it?

Reply:
> @superboyiii could you test it?

Sure

Reply:
> @superboyiii could you test it?

@shargon Due to my test result, the same as @AnnaShaleva said. We need solve it.